### ISSUE TITLE
usage table cannot list service set members

### ISSUE THREAD

**Issue by vita2** [Icinga/icingaweb2-module-director#1122]
the new usage table cannot list service set members.

![service-template-01](https://user-images.githubusercontent.com/25298055/29741916-33d43970-8a76-11e7-95fb-f785c3c07842.png)

![service-template-02](https://user-images.githubusercontent.com/25298055/29741924-5b685fac-8a76-11e7-9417-7299edc9dabf.png)

## Current Behavior
when clicking the link with the amount of set-members i get the following error message:
```
Fatal error: Uncaught exception 'ErrorException' with message 'Creating default object from empty value' in /usr/share/icingaweb2/library/Icinga/Web/Controller/ActionController.php:127 Stack trace: #0 /usr/share/icingaweb2/library/Icinga/Web/Controller/ActionController.php(127): Icinga\Application\ApplicationBootstrap->Icinga\Application\{closure}(2, 'Creating defaul...', '/usr/share/icin...', 127, Array) #1 /usr/share/icingaweb2/library/Icinga/Web/Controller/Dispatcher.php(59): Icinga\Web\Controller\ActionController->__construct(Object(Icinga\Web\Request), Object(Icinga\Web\Response), Array) #2 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response)) #3 /usr/share/icingaweb2/library/Icinga/Application/Web.php(389): Zend_Controller_Front->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response)) #4 /usr/share/icingaweb2/library/Icinga/Application/webrouter.php(104): Icinga\Application\Web->dispatch() #5 in /usr/share/icingaweb2/library/vendor/Zend/Controller/Plugin/Broker.php on line 332
```

## Steps to Reproduce (for bugs)
1. create a service template
2. create services, based on the new template
3. create a service set and add the new services
4. switch back to service templates
5. choose your service template
6. at the usage table, click on set members

## Your Environment
* Director version (System - About): commit da2968a6d3657e20a5695823a80f9f29e6c13e56 master branch
* Icinga Web 2 version and modules (System - About): 2.4.1
* Icinga 2 version (`icinga2 --version`): r2.7.0-1
* Operating System and version: Debian Jessie 8.9 (3.16.0-4-amd64)
* Webserver, PHP versions: nginx 1.6.2-5 + php5-fpm 5.6.30

**Comment by Thomas-Gelf** [comment]
Congratulations, you found a dead link ;-) :1st_place_medal:

**Comment by dgoetz** [comment]
Just as an update as I am not 100% sure, at the moment the members of a service set are counted as objects, but when clicking on it the list will not contain them.

![grafik](https://user-images.githubusercontent.com/3658213/169247354-4d12f51e-500b-45d5-9a87-fdb7dd78e043.png)

![grafik](https://user-images.githubusercontent.com/3658213/169247440-99ab7ecb-70d3-4c18-a9ed-36e4ac9ab0d7.png)

**Comment by Thomas-Gelf** [comment]
This is going to be addressed with #2378
